usr/sbin/p11sak/p11sak.c

```diff
@@ -25,7 +25,7 @@
 #include <sys/types.h>
 #include <pwd.h>
 
-#include <termios.h>
+#include <pin_prompt.h>
```

Comment:
shouldn't this be `#include "pin_prompt.h"` since its a local header file?

Reply:
No. The CFLAGS for p11sak are extended with `-I${srcdir}/usr/lib/common` and therefore the .h files in this directory can be included over the search path. This is the same in all other tools.

Reply:
I know that it works with <...>, otherwise we would see compile errors, but I would still prefer to include local headers using "..." and not <....>. Kind of a code style thing......

Reply:
ok, But why are the CFLAGS extended then? Anyhow, I will change it here and in all other places.

Reply:
Without the extended CFLAGS it would not find it either way. 
The difference between <...> and "..." is pretty much only that "..." first looks in the directory of the current C-file, and then in the include path, whereas <...> ONLY looks in the include path. 
See https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html
Since pin_prompt.h is not in the same directory as p11sak.c, it won't find it without 
`-I${srcdir}/usr/lib/common`

Reply:
ok